src/java_bytecode/java_entry_point.cpp

```diff
@@ -30,7 +29,9 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "java_entry_point.h"
 #include "java_object_factory.h"
 
-//#include "zero_initializer.h"
+#if 0
+#include "zero_initializer.h"
+#endif
```

Comment:
Just remove this, there is no value in keeping it: 1) a header will be included when necessary; 2) we have git.